.circleci/config.yml

```diff
@@ -104,6 +136,14 @@ commands:
       - run:
           name: Run site-controller tests in real cluster
           command: go test -v -count=1 ./cmd/site-controller -use-cluster
+      - run:
+          name: system information at job finish time
+          command: |
+            set +e
+            cat /sys/fs/cgroup/memory/memory.max_usage_in_bytes
+            free -m
+            uptime
```

Comment:
Both `vmstat` and `sar -h -q ALL -r 10` would be nice.  I was already adding a lot of new output to the job, so I tried to refrain from adding too much.  As you say, I'll shelve it for a future PR.

Reply:
I changed my mind :p 

https://app.circleci.com/pipelines/github/skupperproject/skupper/1707/workflows/b575c41f-820a-4d80-a93e-6509a7946479/jobs/8608

I've put both vmstat and sar.  The good thing about vmstat is that the output is in columns, it is easier to follow, while sar has some additional information, and can get more types of information, if required (network, disk, etc).

I'm tempted to leave both there, vmstat every 10s, sar every 60s.  In any case, I don't think it would be good to have any at 1s.  Too much output.  I also re-established the periodic headers and added vmstat timestamps.

Let me know what you think

Reply:
It looks good. But it would be nicer if there were two separate entries in the CI.
One for each executor being monitored (integration and policy).